functions/Export-DbaLogin.ps1

```diff
@@ -56,6 +56,10 @@ function Export-DbaLogin {
     .PARAMETER Confirm
         If this switch is enabled, you will be prompted for confirmation before executing any operations that change state.
 
+    .PARAMETER SetDefaultTempdb
```

Comment:
Thanks so much @MartinHBA ! I actually do like this idea and think it could even be expanded. Please call the property `DefaultDatabase` and let the person specify whichever database. If there's any interest, you can do the same for Language.

I'll update the Pester test once the modification is made.

Reply:
sure, I can do it per your suggestion. Believe it or not I am new to GitHub and not sure about processes. Should I edit also topic name, all descriptions etc?

Reply:
updated, so it should be okay now per your suggestion

Reply:
Thanks so much! No, you don't have to change the title and description along the way 👍 I'll take a closer look when I have a bit more time.

Reply:
Should be by tonight.